### DISCUSSION TITLE
Showcase of procedural DNA animation

### DISCUSSION THREAD

**Discussion by ivan-the-bio-lover-42** [BradyAJohnston/MolecularNodes#601] [Show and tell]
Hello! I have been studying the DNA nodes of molecular nodes, and recently took a look at the node group from an earlier version of the program that allowed you to combine curves based on proximity. Using a modified version of the DNA nodes and this old join curve node, I have made an animation in Blender 3.6 showcasing procedural, dynamic DNA with lots of animation functionality, including enzymatic DNA cutting, and my own little version of DNA unwinding :) Enjoy!

https://github.com/user-attachments/assets/48a68aa2-64c5-4f78-96e4-550fb3ccfa77

**Comment by BradyAJohnston** [comment]
Thanks so much for sharing!
